app/src/interfaces/input-rich-text-html/input-rich-text-html.vue

```diff
@@ -262,11 +264,45 @@ export default defineComponent({
 
 		const internalValue = computed({
 			get() {
+				const url = getPublicURL();
+				if (props.value) {
+					return props.value.replace(
+						new RegExp(
+							`(<[^=]+=")(${escapeRegExp(
+								url
+							)}assets/[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}(?:\\?[^#"]+)?(?:#[^"]*)?)("[^>]*>)`,
+							'gi'
+						),
+						(_, pre, matchedUrl, post) => {
+							if (!matchedUrl.includes('access_token')) {
+								return `${pre}${addTokenToURL(matchedUrl)}${post}`;
+							}
+							return `${pre}${matchedUrl}${post}`;
+						}
+					);
+				}
 				return props.value;
 			},
+
 			set(newValue: string) {
 				if (newValue !== props.value && (props.value === null && newValue === '') === false) {
-					emit('input', newValue);
+					const url = getPublicURL();
+					const remove_token = newValue.replace(
+						new RegExp(
+							`(<[^=]+=")(${escapeRegExp(
+								url
+							)}assets/[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}(?:\\?[^#"]+)?(?:#[^"]*)?)("[^>]*>)`,
+							'gi'
+						),
+						(_, pre, matchedUrl, post) => {
+							const matched = new URL(matchedUrl);
+							const params = new URLSearchParams(matched.search);
+							params.delete('amp;access_token');
+							params.delete('access_token');
+							return `${pre}${matched.origin}${matched.pathname}${params}${post}`;
```

Comment:
We still have to include the configured token when it exists (from `props.imageToken`)